Problem1.java

```diff
@@ -0,0 +1,36 @@
+// Time Complexity : O(n)
+// Space Complexity : O(n) + map space. ie.e O(n) 
+// Did this code successfully run on Leetcode : No
+// Any problem you faced while coding this : No , but the didnt get the approach.
+
+
+// Your code here along with comments explaining your approach
+        // thinking about the brut force.
+        // binary search for missing element if the array is in sorted , i.e. O(log n)
```

Comment:
That wont be efficient if more than one elements are missing.

Reply:
yes, just implemented it and it takes more time. also included the better solution.